synapse/storage/databases/main/cache.py

```diff
@@ -253,6 +253,7 @@ def _invalidate_caches_for_event(
             self.get_applicable_edit.invalidate((relates_to,))
             self.get_thread_summary.invalidate((relates_to,))
             self.get_thread_participated.invalidate((relates_to,))
+            self.get_mutual_event_relations_for_rel_type.invalidate((relates_to,))
```

Comment:
> Another q: should this use `_attempt_to_invalidate_cache`?

I think that would work 👍

`_attempt_to_invalidate_cache` only clears out the local cache but if we're using it in `_invalidate_caches_for_event`, that is already called over replication 🔁

https://github.com/matrix-org/synapse/blob/1a1abdda42551dad3aadc04a169c25f4cc651a2c/synapse/storage/_base.py#L106-L108

Reply:
👍 https://github.com/matrix-org/synapse/pull/13796/commits/4587be1aaaf78b15cfbf124abfd17b9e4d68e164